components/DismissibleMessage.js

```diff
@@ -43,23 +45,32 @@ const DismissibleMessage = ({ LoggedInUser, messageId, children }) => {
   });
 
   const loggedInAccount = data?.loggedInAccount;
-  if (isDismissedLocally || !loggedInAccount || get(loggedInAccount, `settings.${settingsKey}`)) {
+  if (
+    isDismissedLocally ||
+    (!loggedInAccount && !displayForLoggedOutUser) ||
+    get(loggedInAccount, `settings.${settingsKey}`)
+  ) {
     // Don't show message if user is not logged in or if dismissed
     return null;
   }
 
   return children({
     dismiss: () => {
       setDismissedLocally(true);
-      return editAccountSettings({
-        variables: { account: { id: loggedInAccount.id }, key: settingsKey },
-      });
+      window?.localStorage.setItem(settingsKey, 'true');
```

Comment:
Check `lib/local-storage.js` it has helpers to do that safely & cross-browser

```suggestion
      setLocalStorage(settingsKey, 'true');
```

Reply:
Thanks!